code/modules/religion/honorbound/honorbound_trauma.dm

```diff
@@ -45,46 +48,53 @@
 	if(!isliving(clickingon))
 		return
 
-	var/mob/living/clickedmob = clickingon
+	var/mob/living/clicked_mob = clickingon
 	var/obj/item/weapon = honorbound.get_active_held_item()
 
-	if(!honorbound.DirectAccess(clickedmob) && !isgun(weapon))
+	if(!honorbound.DirectAccess(clicked_mob) && !isgun(weapon))
 		return
 	if(weapon?.item_flags & NOBLUDGEON)
 		return
-	if(!honorbound.combat_mode && (HAS_TRAIT(clickedmob, TRAIT_ALLOWED_HONORBOUND_ATTACK) || ((!weapon || !weapon.force) && !LAZYACCESS(modifiers, RIGHT_CLICK))))
+	if(!honorbound.combat_mode && (HAS_TRAIT(clicked_mob, TRAIT_ALLOWED_HONORBOUND_ATTACK) || ((!weapon || !weapon.force) && !LAZYACCESS(modifiers, RIGHT_CLICK))))
 		return
-	if(!is_honorable(honorbound, clickedmob))
+	check_visible_guilt(clicked_mob)
+	if(!is_honorable(honorbound, clicked_mob))
 		return (COMSIG_MOB_CANCEL_CLICKON)
 
+/// Checks a mob for any obvious signs of evil, and applies a guilty reason for each.
+/datum/brain_trauma/special/honorbound/proc/check_visible_guilt(mob/living/attacked_mob)
+	if(ROLE_SYNDICATE in attacked_mob.faction)
+		// as a reminder, ROLE_SYNDICATE is given to obvious and outward syndicates like nuke ops and mobs,
+		// NOT given to traitors. this should be just fine
+		guilty(attacked_mob, "for their misaligned association with the Syndicate!")
+	if(HAS_TRAIT(attacked_mob, TRAIT_CULT_HALO))
```

Comment:
Maybe include wizards and ascended heretics?

Reply:
> Maybe include wizards and ascended heretics?

I second this. Also include Abductors, Nightmares, and Space Ninjas.